webpack/assets/javascripts/react_app/components/common/forms/InputField.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FormControl } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+const InputField = ({ field, disabled, componentClass }) => (
```

Comment:
this makes me sad :panda_face:
What about https://github.com/theforeman/foreman/blob/a68ffcb24319e30c1724de40f1a445f77d86c319/webpack/assets/javascripts/react_app/components/common/forms/InputFactory.js ?

It seems this is the same, but less generalized, could we use `InputFactory` instead?